docs/index-utils.md

```diff
@@ -36,6 +36,13 @@ Converts a collection lookupDocid (in the raw corpus) to a Lucene internal looku
 Converts to a Lucene internal lookupDocid to a collection lookupDocid  (default: 0)
 
 
+```
+-dumpAllDocids outputCompressionFormat
+```
+Dumps all Docids in the index. For non-tweets collection the order is in ascending of String docid;
+For tweets collection the order is in descending of Long tweet id. Please provide the compression format for the output
+
```

Comment:
non-tweets -> non-tweet
what are the options for the compression formats?

Reply:
The options are defined as enum `Compression`